backend/api/controllers/index.js

```diff
@@ -132,7 +118,7 @@ const ResubscriptionErrors = {
 
 const resubscribe = async (request, response) => {
 	try {
-		const user = await getUserFromRequest(request);
+		const user = await User.findByRequest(request);
 
 		const jobs = await getRepeatableJobsByID(user.id);
 		const jobsArray = Object.keys(jobs).map(key => jobs[key]);
```

Comment:
`Object.values(jobs)`

Reply:
Unrelated to this PR. Will refactor at a later stage.